usecases/Concrete/knowledgeGraph/mixture/extractMetadataFromMischung.py

```diff
@@ -0,0 +1,90 @@
+# Updates:
+# This version can accept also excel-files with multiple Rezeptur sheets.
+
+# To-Do: Include xls files. (make sure to have: pip install xlrd) 
+#        Avoid any comma in the output.
+
+# ATTENTION: Doesn't work yet with the most recently added mixture datasets from 16.03.2022!
+
+#----------------------------------------------------------------------------------------------------------
+
+# Read excel with pandas, display with iPython, collect files with glob
+import pandas as pd
+from IPython.display import display
+import glob as g
+import os
+
+# Locate the mixture data (concrete > Data > Mischungen)
+datapath = os.path.realpath(os.path.join(os.path.dirname(__file__), '..', '..', 'Data', 'Mischungen'))
+
+# Displaying setting
+pd.set_option('display.max_rows', None)
+
+# Collect a list of excel-Files in the current folder
+excelsheetlist = g.glob(datapath + '/*xlsx') # + g.glob('./*xls')  Implement later, causes issues right now
+print("There are " + str(len(excelsheetlist)) + " excel-sheets to extract metadata from.")
+print(excelsheetlist)
+
+# Translation: Read in the translation from an excel-file and prepare a function to replace german with english
+translation = pd.read_excel("translation.xlsx")
+translation.columns = ["german", "english"]
+def translate(df):
+    for deutsch, english in translation.itertuples(index=False,name=None):
+        df = df.replace(deutsch,english,regex=True)
+    return df
+    
+# Set up lists to fill inside the loop and build final dataframe from
+list_of_dfs = []
+list_of_excelsheets = []
+
+# main part: extract the metadata for each excelfile and his sheet(s) in the folder
+for excelsheet in excelsheetlist:
```

Comment:
Still not a function, this loop is done in pydoit by definining the dependencies e.g. [here](https://github.com/BAMresearch/ModelCalibration/blob/d60dd8e001e927050ab27907e7be282fe17b49fc/usecases/linearFunction/dodo.py#L29)